src/util/classes.ts

```diff
@@ -23,26 +24,68 @@ export interface Category {
   children?: Category[];
 }
 
-const COLOR_UNCAT = '#CCC';
+const COLOR_GREEN = '#0F0',
+  COLOR_SUPER_GREEN = '#54e346',
+  COLOR_BRIGHT_GREEN = '#A8FC00',
+  COLOR_UNCAT = '#CCC',
+  COLOR_YELLOW = '#FCC400',
+  COLOR_SLIGHTLY_YELLOW = '#f5ff65',
+  COLOR_ORANGE = '#ffba47',
+  COLOR_RED = '#F80';
```

Comment:
Red is orange now? Previous red was `#F33`.

```suggestion
  COLOR_RED = '#F33';
```